pyne/gamma_effects.py

```diff
@@ -0,0 +1,35 @@
+"""Purpose:
```

Comment:
We use numpydoc format.  So you don't need "purpose" and the following lines shouldn't be indented. thanks!